### ISSUE TITLE
[Bug]: Dropsuite / Ninja Saas Backup login blocked

### ISSUE THREAD

**Issue by SLYNETAG** [CyberDrain/Check#85]
### Required confirmations before submitting

- [x] **I can reproduce this issue on the latest released version** of Check.
- [x] **I have searched existing issues** (both open and closed) to avoid duplicates.
- [x] I am **not** requesting general support; this is an actual bug report.

### Issue Description

This URL is a false positive: https://eu-saas-npp[.]backup[.]ninjarmm[.]com/sign_in

<img width="599" height="1238" alt="Image" src="https://github.com/user-attachments/assets/9e978ed3-863f-4eee-b7b5-8e57d5401804" />

### Extension Version

1.0.5

### Rules Version

1.0.6

### Relevant Logs / Stack Trace

```plaintext

```

**Comment by SLYNETAG** [comment]
Same behaviour, though slightly different detection:

<img width="754" height="1311" alt="Image" src="https://github.com/user-attachments/assets/27c85bdf-883f-4da0-ab49-a0981ac37410" />
